### PR TITLE
feat: Add Snackbar component

### PR DESCRIPTION
**Pull request by NickyMeuleman**
Added SnackBar component.
Look at it here: https://streamable.com/k6878

TODO: connect to apollo-error-link

### CODE REVIEW

**Comment by pantharshit00** [comment]
Nice work! to be clear error link is global need of integration. So I am merging this

**Comment by pantharshit00** [comment]
I will be connecting this to link State in another commit. MERGING....

**Comment by pantharshit00** [comment]
Also I have commentizen setup if you are not using it.

File: src/components/UserLayout.js [hunk 1]
```diff
@@ -28,6 +28,9 @@ const Content = styled.div`
 `
 
 export default class Layout extends Component {
+  state = {
+    isSnackBarOpen: true
+  }
```
Comment (NickyMeuleman) [inline src/components/UserLayout.js]: Forgot to remove this when my testing was done. This piece can go.

**Pull request merged**